guides/source/form_helpers.md

```diff
@@ -516,22 +535,30 @@ Outputs a select box like:
 </select>
 ```
 
-Notice that the appropriate option was automatically marked `selected="selected"`. Since this select box was bound to a model, we didn't need to specify a `:selected` argument!
+The only difference is that the selected option will be found in `params[:person][:city]` instead of `params[:city]`.
+
+Notice that the appropriate option was automatically marked `selected="selected"`. Since this select box was bound to an existing `@person` record, we didn't need to specify a `:selected` argument.
+
+### Selecting Time Zone
```

Comment:
I like how you changed this from "Time Zone and Country Select" to "Selecting a Time Zone", maybe we can do the same with " the header above this "Select Boxes and Model Objects". What do you think about something more descriptive like "Binding Select Boxes to Model Objects" ? Sorry I for some reason could not drop this comment on that line and had to comment it here where you made a change.

Reply:
done!